<!--
Copyright (c) 2018 Pilz GmbH & Co. KG

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU Lesser General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU Lesser General Public License for more details.

You should have received a copy of the GNU Lesser General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->

<launch>
  <arg name="safety_hw" default="pss4000" />
  <arg name="api_spec_file" default="$(find prbt_hardware_support)/config/modbus_api_spec_$(arg safety_hw).yaml" />

  <node required="true" ns="prbt" pkg="prbt_hardware_support" type="modbus_brake_test_announcer_node" name="modbus_brake_test_announcer_node" output="screen">
    <rosparam ns="api_spec" command="load" file="$(arg api_spec_file)" />
  </node>

</launch>
